common/model/BUILD

```diff
@@ -1,10 +1,77 @@
 package(default_visibility = ["//visibility:public"])
 
 cc_library(
-    name = "transaction",
-    srcs = ["transaction.c"],
-    hdrs = ["transaction.h"],
-    deps = [
-        "//common/trinary:tryte_long",
-    ],
-)
+    name="transaction",
+    srcs=["transaction.c"],
+    hdrs=["transaction.h"],
+    deps=[
+        "//common/trinary:trit_long",
+        "//common/trinary:trit_array"
+    ], )
+
+cc_library(
+    name="transaction_1",
+    srcs=["transaction.c"],
+    hdrs=["transaction.h"],
+    deps=[
+        "//common/trinary:trit_long",
+        "//common/trinary:trit_array_1"
+    ], )
+
+cc_library(
+    name="transaction_4",
+    srcs=["transaction.c"],
+    hdrs=["transaction.h"],
+    deps=[
+        "//common/trinary:trit_long",
+        "//common/trinary:trit_array_4"
+    ], )
+
+cc_library(
+    name="transaction_5",
+    srcs=["transaction.c"],
+    hdrs=["transaction.h"],
+    deps=[
+        "//common/trinary:trit_long",
+        "//common/trinary:trit_array_5"
+    ], )
+
+cc_library(
+    name="cpptransaction",
+    srcs=["cpptransaction.cc"],
+    hdrs=["cpptransaction.h"],
+    deps=[
+        "//common/model:transaction",
+        "//common/trinary:trit_long",
+        "//common/trinary:trit_array"
+    ], )
+
+cc_library(
```

Comment:
There's no need for the various `_{1,4,5}` targets. We're already testing `transaction_{1,4,5}` elsewhere. So one target should be sufficient